source/_integrations/openai_conversation.markdown

```diff
@@ -16,6 +16,6 @@ The OpenAI integration adds a conversation agent powered by [OpenAI](https://www
 
 This conversation agent is unable to control your house. It can only query information that has been provided by Home Assistant. To be able to answer questions about your house, Home Assistant will need to provide OpenAI with the details of your house, which include areas, devices and their states.
 
-This integration requires an API key to use, [which you can generate here.](https://beta.openai.com/account/api-keys).
+This integration requires an API key to use, [which you can generate here](https://beta.openai.com/account/api-keys).
```

Comment:
I had actually done that on purpose. It looks weird to have a single black character after a hyperlink.

Reply:
Both are done... we have no strict rules around it and boils down to personal preference. Since the integration is by balloob, I'll go with his preference right now.